sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -499,9 +528,50 @@ private void initialize() {
                 mUncaughtExceptionHandler = null;
             }
         } else {
+
+            /* Register Java crash handler. */
             mUncaughtExceptionHandler = new UncaughtExceptionHandler();
             mUncaughtExceptionHandler.register();
-            final File logFile = ErrorLogHelper.getLastErrorLogFile();
+
+            /* Convert minidump files to App Center crash files. */
+            for (File logFile : ErrorLogHelper.getNewMinidumpFiles()) {
+
+                /* Create missing files from the native crash that we detected. */
+                AppCenterLog.debug(LOG_TAG, "Process pending minidump file: " + logFile);
+                File dest = new File(ErrorLogHelper.getPendingMinidumpDirectory(), logFile.getName());
+                NativeException nativeException = new NativeException();
+                Exception modelException = new Exception();
+                modelException.setType("minidump");
+                modelException.setWrapperSdkName(Constants.WRAPPER_SDK_NAME_NDK);
+                modelException.setStackTrace(dest.getPath());
+                ManagedErrorLog errorLog = new ManagedErrorLog();
+                errorLog.setException(modelException);
+                errorLog.setTimestamp(new Date(logFile.lastModified()));
+                errorLog.setFatal(true);
+                errorLog.setId(UUID.randomUUID());
+
+                /* TODO The following properties are wrong. */
```

Comment:
Is this something that needs to be fixed before merging, then?

Reply:
Not really